manifests/f/FACEITLTD/FACEITClient/2.0.41/FACEITLTD.FACEITClient.installer.yaml

```
@@ -0,0 +1,23 @@
+# Automatically updated by the winget bot at 2025/Mar/05
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: FACEITLTD.FACEITClient
+PackageVersion: 2.0.41
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: machine
+InstallModes:
+- silent
+InstallerSwitches:
+  Silent: --silent
+  SilentWithProgress: --silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://faceit-client.faceit-cdn.net/release/FACEIT-setup-latest.exe
+  InstallerSha256: fce5e1d0c13cd471bf4c8a9611513e40eccc36323d7cfe85a6f1a419b8e1fb66

```

```suggestion
  InstallerSha256: A6DF926B624029776EA8010A271FCB4DD46FA7285316E282A1F43DC883B17B85
```

(Automated response - build 994.)